lightningd/opening/opening_wire.csv

```diff
@@ -29,27 +29,26 @@ opening_funder,0,funding_satoshis,8
 opening_funder,8,push_msat,8
 opening_funder,16,feerate_per_kw,4
 opening_funder,20,max_minimum_depth,4
+opening_funder,24,change_satoshis,u64
+opening_funder,32,change_keyindex,u32
+#include <lightningd/utxo.h>
+opening_funder,0,num_inputs,u16
+opening_funder,0,inputs,num_inputs*struct utxo
+opening_funder,0,bip32_len,u16
+opening_funder,0,bip32_seed,bip32_len*u8
 
-# Reply asks for txid of funding transaction.
+# This gives their sig, means we can broadcast tx: we're done.
 opening_funder_reply,101
-opening_funder_reply,0,local_fundingkey,33
+opening_funder_reply,0,their_config,struct channel_config
+opening_funder_reply,36,first_commit_sig,secp256k1_ecdsa_signature
+opening_funder_reply,100,crypto_state,struct crypto_state
+opening_funder_reply,244,revocation_basepoint,33
+opening_funder_reply,277,payment_basepoint,33
+opening_funder_reply,310,delayed_payment_basepoint,33
+opening_funder_reply,343,their_per_commit_point,33
+opening_funder_reply,376,minimum_depth,4
 opening_funder_reply,0,remote_fundingkey,33
```

Comment:
No, in fact we never use them.  They're vestigial from the output format from the spec.  Should really clean them all up.  I think we could put ,, there in fact....